sql-plugin/src/main/scala/com/nvidia/spark/rapids/shuffle/RapidsShuffleIterator.scala

```diff
@@ -16,15 +16,14 @@
 
 package com.nvidia.spark.rapids.shuffle
 
-import java.util.concurrent.LinkedBlockingQueue
-
-import scala.collection.mutable
-
 import ai.rapids.cudf.{NvtxColor, NvtxRange}
 import com.nvidia.spark.rapids._
+import java.util.concurrent.LinkedBlockingQueue
+import scala.collection.mutable
```

Comment:
Why were these imports moved around?  Normally scala imports are separate per [the recommended style import layout.](https://github.com/NVIDIA/spark-rapids/blob/branch-0.2/docs/dev/idea-code-style-settings.md)

Reply:
Short answer, I hadn't applied the layout settings correctly, and somehow `mvn verify` is not picking this up, even with the `ImportOrderChecker` scalatyle rule.